demo/qat-init.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# shellcheck disable=SC2154
+for dev in $(for pf in $ENABLED_QAT_PF_PCIIDS; do lspci | grep "$pf" | cut -f 1 -d ' '; done); do
```

Comment:
maybe disable `CONFIG_LSPCI_TEXT` and `CONFIG_CUT` and just?
```suggestion
for dev in $(for pf in $ENABLED_QAT_PF_PCIIDS; do lspci -n| grep -o -e "$pf" -e "^\S*"; done); do
```

Reply:
Umm, Idk why, but regex does not work properly to me.
^\S* does not return the things before a whitespace..

Reply:
Ah,
I found the reason...
idk why, but we need to put
\^\\S\*